Assets/Scripts/Player/Dash.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Dash : MonoBehaviour
+{
+    [SerializeField] private float _dashPower;
+    private Rigidbody _rb;
+    private Quaternion _direction;

```

Comment:
Why are you storing this?
Why retrieve this every single frame, when you can also just retrieve it only when the player actually dashes?